cdserver/features/monitoring_and_control/step_definitions/monitoring_and_control_steps.rb

```diff
@@ -0,0 +1,20 @@
+# CucumberJS step definitions for healthcheck feature test
+
+Given(/^the capdash system is running$/) do
+  Rails.application.initialized?
+end
+
+Given(/^StatsD is running$/) do
+  StatsD.present? && File.open('log/statsd.log', 'w+').close
+end
+
+When(/^I navigate to the url (.*)$/) do |url|
+  get url
+end
+
+Then(/^I should see the log '(.*?)'$/) do |expected_message|
```

Comment:
if you check before and after, and the most recent number of the log was x, then afterwards it should be x+1